### PR TITLE
Adds context menu options for directly executing wgt package file + fixes log issues

### PR DESCRIPTION
**Pull request by prate3k**
**Changes**
- Improve plugin usability by adding context menu option for installing/debugging `.wgt` package file
    - New Flow addition : 
        - Right click on any `.wgt` package file, and choose "Tizen TV: Launch / Debug Application"
        - Specify target device to run on (same step as earlier)
        - Enter package id (new step addition). (i.e. package id specified in `config.xml` file)
        - Done.
- Fixes
    - Updates `package-lock.json` to make it in sync with `package.json`.
    - Invalid information in logs (i.e `start(${value})`) + typos.

**Screenshot**
Screenshot 2022-03-06 at 6 21 55 PM

### CODE REVIEW

**Comment by prate3k** [comment]
@taronlee @SejoongDeJang @wb0716 Review please. :)

File: lib/launchApplication.js [hunk 1]
```diff
@@ -29,12 +32,29 @@ module.exports = async function launchApplication(isDebug) {
...
 
-    let webApp = TVWebApp.openProject(vscode.workspace.rootPath);
-    if (webApp == null) {
-        logger.log('launchApplication:web App is null');
-        return;
+    let results = await controller.start();
+    
+    let webApp;
+    if (wgtFilePath) {
+        if (!fs.existsSync(wgtFilePath)) {
```
Comment (taronlee) [inline lib/launchApplication.js:40]: I think it would be better to use `async/await` instead of a sync call

**Review by taronlee — DISMISSED** [review]

File: lib/launchApplication.js [hunk 2]
```diff
@@ -29,12 +32,29 @@ module.exports = async function launchApplication(isDebug) {
         });
     }
 
-    let results = await controller.start();
+    if (wgtFilePath) {
```
Comment (pwsses) [inline lib/launchApplication.js]: I think input package id directly every time when launch with wgt file is not very usability for user.
how about get package id from config.xml in wgt file.
I mean, you unzip wgt file into temp folder to get confilg.xml file and delete it.

Comment (SejoongDeJang) [reply lib/launchApplication.js]: @prate3k Refer to followed package.
https://stackoverflow.com/questions/39705209/node-js-read-a-file-in-a-zip-without-unzipping-it
You can get the `config.xml` of the `.wgt` without unzipping.

File: lib/launchApplication.js (hunk 1 above)
Comment (prate3k) [reply lib/launchApplication.js:40]: Yes, I think I saw some references to `fs.readFileSync` inside repo hence took this approach to maintain consistency.

File: lib/launchApplication.js (hunk 2 above)
Comment (prate3k) [reply lib/launchApplication.js]: Thanks for the suggestion :) Checking this.

Comment (prate3k) [reply lib/launchApplication.js]: Found out a way to use existing `decompress` and `xml2js` module for extracting application id. I've pushed the changes. Can you please re-review.

**Review by pwsses — APPROVED** [review]

**Pull request merged**